src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -61,30 +80,59 @@ type LnFeeCalculator = {
   max(amount: Satoshis): Satoshis
 }
 
+type PayInvoiceResult = {
+  roundedUpFee: Satoshis
+}
+
 interface ILightningService {
   isLocal(pubkey: Pubkey): boolean | LightningServiceError
+
+  defaultPubkey(): Pubkey
+
+  lndFromPubkey(pubkey: Pubkey): AuthenticatedLnd | LightningServiceError
+
   registerInvoice(
     registerInvoiceArgs: RegisterInvoiceArgs,
   ): Promise<RegisteredInvoice | LightningServiceError>
+
   lookupInvoice({
     pubkey,
     paymentHash,
   }: {
     pubkey: Pubkey
     paymentHash: PaymentHash
   }): Promise<LnInvoiceLookup | LightningServiceError>
+
   lookupPayment({
     pubkey,
     paymentHash,
   }: {
     pubkey: Pubkey
     paymentHash: PaymentHash
   }): Promise<LnPaymentLookup | LightningServiceError>
+
   cancelInvoice({
     pubkey,
     paymentHash,
   }: {
     pubkey: Pubkey
     paymentHash: PaymentHash
   }): Promise<void | LightningServiceError>
+
+  payInvoice({
+    paymentHash,
+    rawRoute,
+    lndAuthForRoute,
+    decodedInvoice,
+    maxFee,
+    timeoutMs,
+  }: {
+    paymentHash: PaymentHash
+    rawRoute: RawRoute | null
+    lndAuthForRoute: AuthenticatedLnd | null
+    decodedInvoice: LnInvoice
+    milliSatsAmount: MilliSatoshis
+    maxFee: Satoshis
+    timeoutMs?: MilliSeconds
```

Comment:
I guess this was done previous to optional params conversation, maybe we just need a default value here

Reply:
Yup, removed now